chromium_src/chrome/browser/resources/tab_search/auto_tab_groups/auto_tab_groups_page.css

```diff
@@ -166,3 +167,60 @@
   padding: var(--leo-spacing-l);
   box-sizing: border-box;
 }
+
+.enable-tab-focus-wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  box-sizing: border-box;
+  padding: var(--leo-spacing-m);
+  gap: var(--leo-spacing-m);
+}
+
+.enable-tab-focus-header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+  width: 100%;
+  gap: var(--leo-spacing-m);
+  box-sizing: border-box;
+}
+
+.enable-tab-focus-content-wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  box-sizing: border-box;
+  border-radius: var(--leo-radius-m);
+  background-color: var(--leo-color-neutral-10);
+  overflow: hidden;
+}
+
+.enable-tab-focus-illustration {
+  background-image: url(//resources/brave/images/tab_organizer_illustration.svg);
+  background-size: cover;
+  width: 100%;
+  height: 174px;
```

Comment:
is this implied by the height of the SVG? i.e it should maintain its aspect ratio right, so maybe we can just set `width: 100%` here.

Reply:
If the `align-items` from above works, we can probably remove this too

Reply:
Needed to keep the `height` here, but `align-items: stretch` worked nicely.

Reply:
Interesting - doesn't this mean we chop off bits of the image at certain aspect ratios (because we always set the height to the same thing)